scripts/trelloEngWorkInUrgencyDashboard.js

```diff
@@ -3,7 +3,8 @@ const TRELLO_CREDENTIALS_HELPER_FILE = "https://docs.google.com/document/d/1Hwae
 const LIST_IDS = [
     "65e07588569c27c0664edf17",
     "65d8c6c2bdf8fb6ece5c3288",
-    "65eb3c3f18360ab17df520fd"
+    "65eb3c3f18360ab17df520fd",
+    "65f86bf9394966aa2a03379f"
```

Comment:
@eripoche These ids are very hard to understand what they refer to ? 
Would it make sense to explict this value ? (ex: a link to the trello list or first card, the name of the squad ...)

Reply:
@EtienneLeco Good idea, I added a comment for each https://github.com/360Learning/360learning.github.io/pull/28/commits/f7795ccf559da5d276014d4ecec2b2ba4fbb2b56. Thanks!

Reply:
@eripoche @EtienneLeco Hi !
> These ids are very hard to understand what they refer to ?

That was the point, since the GH repo is public 😅
I believe we don't want to publicly expose the squad names.
Hence the warning
> [tech warning] this repo is public, do not add any sensitive information in there. That’s why squad names are fetched and not hard coded.

 in the [practice](https://360learning.atlassian.net/wiki/spaces/360LEARNIN/pages/195100830/ENG+Practices+-+Week+3+-+Working+in+a+Product+Squad#How-%3F).
Wdyt ?

Reply:
@MartinBB360 So just the Trello link?

Reply:
@eripoche
> The squad name is considered sensitive information?

I guess so, but it's debatable.
Keep only the trello link is fine to me !

Reply:
@eripoche Thank you 🙏